hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/RegionServerCoprocessorHost.java

```diff
@@ -166,6 +169,16 @@ public void call(RegionServerObserver observer) throws IOException {
     });
   }
 
+  public void preReplicationSinkBatchMutate(AdminProtos.WALEntry walEntry, Mutation mutation)
```

Comment:
The coprocessor API is symmetric. Whenever there is a preXXX action, giving the user control before a thing happens, there must also be a postXXX action, letting the user get control after the thing happened. There can be case by case exceptions when no postXXX condition actually exists, like after a shutdown.

Reply:
Added postXXX()